.gitignore

```diff
@@ -1,8 +1,6 @@
-.DS_Store
```

Comment:
.idea and *.iml must be completely ignored;
moreover, extra gradlew in `MagicTextView` is redundant as well